src/server.py

```diff
@@ -1,26 +1,19 @@
-# _*_ coding: utf-8 _*_
+# _*_ Coding: utf-8 _*_
 
 import socket
-import io
-import pdb
+import sys
 import os.path
 import os
+import io
+import pdb
 
 RSC = u'webroot{}'
 
-def server():
-    server = socket.socket(socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_TCP)
-    address = ('127.0.0.1', 5050)
-    server.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-    server.bind(address)
-    return server
-
 
-def listen_to(server):
-    try:
-        server.listen(1)
-        conn, addr = server.accept()
-        buffer_length = 8
+def asynch(conn, address):    
+    """Tell the StreamServer what to do with incoming data."""
+    try:       
+        buffer_length = 16
```

Comment:
Lines 21/22 and 24/25 below are really duplicates.  There's no serious penalty to concatenating an empty string to your accumulated message, so you can eliminate 21/22 entirely.